pyproject.toml

```diff
@@ -1,10 +1,35 @@
 [build-system]
+requires = [
+         "setuptools",
+         "setuptools_scm",
+         "wheel",
+         "oldest-supported-numpy",
+         "extension-helpers"
```

Comment:
Are we actually using anything from extension helpers?

Reply:
No.

Reply:
Done.